examples/AspNetCore/Startup.cs

```diff
@@ -92,15 +93,33 @@ public void ConfigureServices(IServiceCollection services)
                         }));
                     break;
                 default:
-                    services.AddOpenTelemetryTracing((builder) => builder
-                        .AddAspNetCoreInstrumentation()
-                        .AddHttpClientInstrumentation()
-                        .AddConsoleExporter());
+                    // Adding OtelTracing. Prepares to accept more configs from DI.
```

Comment:
@tekian Please take a look. (In context of https://github.com/open-telemetry/opentelemetry-dotnet/issues/1394)